src/Microsoft.Diagnostics.ExtensionCommands/Host/CommandExtensions.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Diagnostics.DebugServices;
+using Microsoft.Diagnostics.ExtensionCommands.Output;
+using Microsoft.Diagnostics.Runtime;
+
+namespace Microsoft.Diagnostics.ExtensionCommands
+{
+    public static class CommandExtensions
+    {
+        public const string DacTableSymbol = "g_dacTable";
+        public const string DebugHeaderSymbol = "DotNetRuntimeDebugHeader";
+
+        /// <summary>
+        /// Displays the special diagnostics info header memory block (.NET Core 8 or later on Linux/MacOS)
+        /// </summary>
+        public static void DisplaySpecialInfo(this CommandBase command, string indent = "")
+        {
```

Comment:
Some of these relate to a runtime, some of them to a module, and some of them to a target. Isn't it better to get them to get them on those?

Reply:
They are called by multiple commands and they need the services that the CommandBase provides. If they were IModule  or ITarget extension functions, the code couldn't get the same set of services that the original command was invoked with (like current runtime, thread or target).  I look at them as shared command code instead of ITarget or IModule extensions.

Reply:
What services to they really need? I largely think these should be methods that return info as string, the fact they are logged to a console should be a concern of the caller.